antspymm/mm.py

```diff
@@ -4011,6 +4013,7 @@ def resting_state_fmri_networks( fmri, fmri_template, t1, t1segmentation,
   outdict['ssnr'] = slice_snr( corrmo['motion_corrected'], csfAndWM, gmseg )
   outdict['dvars'] = dvars( corrmo['motion_corrected'], gmseg )
   outdict['high_motion_count'] = (rsfNuisance['FD'] > 0.5 ).sum()
+  outdict['high_motion_pct'] = (rsfNuisance['FD'] > 0.5 ).sum() / rsfNuisance.shape[0]
```

Comment:
I don't think 0.5 is the same here as in Powers papers .... either (a) this needs to be a variable or (b) we need to adjust the FD calculation to best match ">0.5 is bad" standards ... I would guess that this threshold will lead to roughly half the volumes being labeled "bad" ... for this reason, I think this should be removed until the above issues are resolved.

Reply:
ok - that's fine - then pass 'FD_threshold' as a variable s.t. we can determine its appropriate setting later ...